### PR TITLE
Fixed LP bug #1595263 for server variables

### PR DESCRIPTION
**Pull request by hrvojem**
moved unique_checks description to known issues as it shoud gain better
visibility there

### CODE REVIEW

File: doc/source/tokudb/tokudb_variables.rst [hunk 1]
```diff
@@ -4,452 +4,1525 @@
...
+  :conf: Yes
+  :scope: Global
+  :dyn: No
+  :vartype: Numeric
+  :range: 0 - 1024
+  :default: 0
+
+This variable defines the number of threads for the checkpoint worker thread
+pool. This pool is used to serialize, compress and write nodes cloned during
+checkpoint. It uses old algorithm to set pool size to 4 / num_cpu_threads.
```
Comment (george-lorch) [inline doc/source/tokudb/tokudb_variables.rst]: This variable defines the number of threads for the cachetable worker thread pool. This pool is used to perform node prefetches, and to serialize, compress, and write nodes during cachetable eviction.
The default value of 0 calculates the pool size to be num_cpu_threads * 2.

File: doc/source/tokudb/tokudb_variables.rst [hunk 2]
```diff
@@ -4,452 +4,1525 @@
...
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Session, Global
+  :dyn: Yes
+  :vartype: Boolean
+  :default: OFF
+
+Disables checkpointing when true. Session variable but acts like a global, any
+session disabling checkpointing disables it globally. Special purpose only, do
```
Comment (george-lorch) [inline doc/source/tokudb/tokudb_variables.rst]: If a session sets this lock and disconnects or terminates for any reason, the lock will not be released.

File: doc/source/tokudb/tokudb_variables.rst [hunk 3]
```diff
@@ -4,452 +4,1525 @@
...
+  :version 5.6.26-74.0: Implemented
+  :cli: Yes
+  :conf: Yes
+  :scope: Session, Global
+  :dyn: Yes
+  :vartype: Numeric
+  :range: 2-16384
+  :default: 16
+
+This variable controls the Fractal Tree fanout.
```
Comment (george-lorch) [inline doc/source/tokudb/tokudb_variables.rst]: The fanout defines the number of 'pivot' keys or child nodes for each internal tree node.

Changing the value of :variable:`tokudb_fanout` only affects subsequently created tables and indexes. The value of this variable cannot be changed for an existing table/index without a dump and reload.

**Review by george-lorch — CHANGES_REQUESTED** [review]

**Comment by hrvojem** [comment]
Fixed/pushed version which addresses the comments

**Review by george-lorch — APPROVED** [review]
TokuDB stuff looks good

*edit : Oops, no, wait...*

File: doc/source/tokudb/tokudb_variables.rst [hunk 4]
```diff
@@ -4,472 +4,1531 @@
...
+  :scope: Global
+  :dyn: No
+  :vartype: Numeric
+  :range: 0 - 1024
+  :default: 0
 
-Fractal tree internal and leaf nodes default to 4,194,304 bytes (4 MB). The session variable :variable:`tokudb_block_size` controls the target uncompressed size of these nodes.
+This defines the number of threads for the checkpoint worker thread pool. This
+pool is used to serialize, compress and write nodes cloned during checkpoint.
+Uses old algorithm to set pool size to ``num_cpu_threads/4``.
```
Comment (george-lorch) [inline doc/source/tokudb/tokudb_variables.rst]: "Default of 0 uses old algo..."

File: doc/source/tokudb/tokudb_variables.rst [hunk 5]
```diff
@@ -4,472 +4,1531 @@
...
+  :dyn: No
+  :vartype: Numeric
+  :range: 0 - 1024
+  :default: 0
 
-.. variable:: tokudb_lock_timeout_debug
+This variable defines the number of threads for the client operations thread
+pool. This pool is used to perform node maintenance on over/undersized nodes
+such as message flushing down the tree, node splits, and node merges. Uses old
+algorithm to set pool size to 1 * num_cpu_threads.
```
Comment (george-lorch) [inline doc/source/tokudb/tokudb_variables.rst]: Default value of 0 uses old ...

**Review by george-lorch — CHANGES_REQUESTED** [review]

**Review by george-lorch — APPROVED** [review]

**Pull request merged**